mods/sgssb/moves.js

```diff
@@ -615,7 +615,8 @@ exports.BattleMovedex = {
 		id: "aquasubscribe",
 		name: "Aqua Subscribe",
 		priority: 1,
-		self: {
+		secondary: {
```

Comment:
Oh yeah.... Just using self works though haha but I made it secondary for your sake